test/e2e-tests.sh

```diff
@@ -28,42 +28,62 @@ source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/e2e-tests.s
 
 # Helper functions.
 
-# Setup the Knative environment for running tests
+readonly EVENTING_CONFIG="config/"
+
+readonly IN_MEMORY_CHANNEL_PROVISIONER_CONFIG="config/provisioners/in-memory-channel/in-memory-channel.yaml"
+
+readonly GCP_PUBSUB_PROVISIONER_CONFIG="contrib/gcppubsub/config/gcppubsub.yaml"
+readonly GCP_PUBSUB_TEMP_CONFIG="$(mktemp)"
```

Comment:
I suggest adding a newline and a comment here like in these two to make it clear the difference:

```
# GCP PubSub config template.
readonly GCP_PUBSUB_PROVISIONER_CONFIG...
# Real GCP PubSub config, generated from the template. 
readonly GCP_PUBSUB_TEMP_CONFIG...
```

I would even drop TEMP from the name, as it makes me think this is a temporary config. What about GCP_PUBSUB_CONFIG_TEMPLATE and GCP_PUBSUB_CONFIG ? (may even dropping PROVISIONER from IN_MEMORY_CHANNEL_PROVISIONER_CONFIG for simplicity and name consistency)

Reply:
Done.

Reply:
Looks like the changes were not pushed? I still see the old names and no comments.

Reply:
Sorry not sure why it's missed. Done this time.